src/styles/main.scss

```diff
@@ -0,0 +1,117 @@
+$mainTxtColor: #060b35;
+$secondaryTxtColor: #616070;
+$cardMainCol: #fff;
+$cardBorder: #f3f3f3;
+$cardBtnColor: #00acdc;
+$gap: 16px;
+$mainFont: "Roboto", sans-serif;
+
+@mixin base($color, $borderCol) {
+  background-color: $color;
+  border: 1px solid $borderCol;
+  border-radius: 5px;
+}
+
+@mixin font($weight, $size, $line: $size) {
+  font-size: $size;
+  line-height: $line;
+  font-weight: $weight;
+}
+
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+  margin: 0;
+}
```

Comment:
you can use that in index.scss file, after creating files for card and stars

Reply:
I will move css blocks to separate files and leave those lines in main.scss, under imports. ( I usually not using "index" name for scss) :)